api/utils/main.go

```diff
@@ -478,6 +478,10 @@ func UserInputCredentials() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if len(bytePassword)==0{
+		fmt.Println("Secret is empty!")
```

Comment:
You can use here logger.err()
Also when the password is empty the program should terminate and no file write had happened

Reply:
Shouldn't we give the user another chance to enter the secrets?

Reply:
Not sure if it affects usability 🤔

Reply:
Okay let's get user another chance